bedrock/base/templates/includes/protocol/footer/footer-newsletter.html

```diff
@@ -0,0 +1,21 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+<div class="moz24-newsletter-container">
+  <div class="moz24-newsletter-wrapper">
+    <div class="moz24-newsletter-info">
+      <img loading="lazy" class="moz24-newsletter-image" src="{{ static('img/logos/mozilla/footer-refresh/symbol.svg') }}" alt="" width="40" height="48">
+      <p>{{ ftl('footer-refresh-find-out-about') }}</p>
+    </div>
+
+    <div class="moz24-newsletter">
+      {{ email_newsletter_form(
+        newsletters='knowledge-is-power',
```

Comment:
**Issue:** The `knowledge-is-power` newsletter is going away (see: https://github.com/mozilla/bedrock/issues/15075). should this be replaced by either the Firefox newsletter, or the Mozilla Foundation newsletter? I'm not sure if we'd also need to change some of the newsletter copy or not then, but thought I should flag.

Reply:
Confirmed with Stephanie that we will be using `mozilla-and-you` newsletter instead

Reply:
Updated.